### PR TITLE
Run one single external container in case of hostnetwork

### PR DESCRIPTION
**Pull request by fedepaol**
When running hostnetworked, frr on metallb side squashes the ip and
configures one single neighbor. Here we add one single container in case
tests are running in hostnetwork mode.

### CODE REVIEW

File: e2etest/bgp_test.go [hunk 1]
```diff
@@ -72,30 +72,28 @@ type containerConfig struct {
...
+	ibgpContainerConfig := containerConfig{
+		name: frrEBGP,
+		nc: frrconfig.NeighborConfig{
+			ASN:      IBGPAsn,
+			Password: "ebgp-test",
+		},
+		rc: frrconfig.RouterConfig{
+			ASN:      EBGPAsn,
+			BGPPort:  180,
+			Password: "ebgp-test",
```
Comment (msherif1234) [inline e2etest/bgp_test.go:96]: if its ibgp container why its called frrEBGP and ASN should be the same for router and nbr ?

Comment (fedepaol) [reply e2etest/bgp_test.go:96]: Because I mismatched the names :-)

File: e2etest/bgp_test.go [hunk 2]
```diff
@@ -72,30 +72,28 @@ type containerConfig struct {
...
-			rc: frrconfig.RouterConfig{
-				ASN:      IBGPAsn,
-				BGPPort:  179,
-				Password: "ibgp-test",
-			},
+	ebgpContainerConfig := containerConfig{
+		name: frrIBGP,
+		nc: frrconfig.NeighborConfig{
+			ASN:      IBGPAsn,
+			Password: "ibgp-test",
```
Comment (msherif1234) [inline e2etest/bgp_test.go:79]: name frrIBGP ? and ASN need to different maybe we should just call them ASN1 and ASN2 ?

Comment (fedepaol) [reply e2etest/bgp_test.go:79]: This is the very same configuration we had in place before, I just cut the slice in two elements. Apart from naming the elements, I think everything else does not belong to this PR (or at least to this commit).

**Comment by msherif1234** [comment]
LGTM

**Comment by oribon** [comment]
LGTM

**Pull request merged**